Resources/doc/config.rst

```diff
@@ -405,9 +405,10 @@ Otherwise you will get a *auth failed* exception.
             connections:
                 default:
                     server: "mongodb://localhost:27017"
-                    username: someuser
-                    password: somepass
-                    authSource: db_you_have_access_to
+                    options:
+                        username: someuser
+                        password: somepass
+                        db: db_you_have_access_to
```

Comment:
Yes, the documentation will be correct for master. The documentation was incorrect, even before this was changed in master. 3.2 also has `username`, `password` and other options under an `options` key.